ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -68,6 +83,17 @@ public override void AddView(RichTextBlock parent, DependencyObject child, int i
             }
 
             parent.Blocks.OfType<Paragraph>().First().Inlines.Insert(index, inlineChild);
+
+            // Initialize ImportantForAccessibility for the child.
+            if (child is UIElement childUIElement)
+            {
```

Comment:
Is this specific to text, or is there some other way we can move this into a base class so all views can participate in ImportantForAccessibility?

Reply:
@rozele This is specific to RIchTextBlock. I am looking at the other views but they have their own patterns. Since the logic is encapsulated into AccessibilityHelper, the code added to AddView() should be minimal.

---
In reply to: [166428781](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r166428781) [](ancestors = 166428781)